docs/testing.md

```diff
@@ -61,50 +61,25 @@ The feature files are located in the "tests/acceptance/features" subdirectories.
 see [available settings](#available-settings-to-be-set-by-environment-variables) for further setup if needed
 
 ### with OCIS backend
+- [setup and build Phoenix]({{< ref "building.md" >}})
 
-- clone and build [ocis](https://github.com/owncloud/ocis)
-- clone the [testing-app](https://github.com/owncloud/testing), it's needed to have the skeleton folder for the tests
+#### the quick way (all automated)
+1. run `yarn run test-requirements:ocis` to install, configure and run all ocis requirements
+2. run `yarn run acceptance-tests-ocis <feature-files-to-test>` to run the tests, the feature files are located in the "tests/acceptance/features" subdirectories.
+3. after the tests run `yarn run docker-kill` to stop all created docker containers
 
-- Run ldap server and redis server using docker
+#### the manual way (e.g. to run from an existing ocis location)
+1. clone and build [ocis](https://github.com/owncloud/ocis)
+2. run `yarn run testing-app` to get the [testing-app](https://github.com/owncloud/testing), it's needed to have the skeleton folder for the tests
+3. Run ldap server and redis server using docker
     ```sh
-    docker run --hostname ldap.my-company.com \
-    -e LDAP_TLS_VERIFY_CLIENT=never \
-    -e LDAP_DOMAIN=owncloud.com \
-    -e LDAP_ORGANISATION=ownCloud \
-    -e LDAP_ADMIN_PASSWORD=admin \
-    --name docker-slapd \
-    -p 127.0.0.1:389:389 \
-    -p 636:636 -d osixia/openldap
-
-    docker run -e REDIS_DATABASES=1 -p 6379:6379 -d webhippie/redis:latest
-    ```
+    yarn run ldap-server
 
-- create a new phoenix config.json file.
-
-  Here an example that should work when running every service on localhost
-  ```
-  {
-    "server": "https://localhost:9200",
-    "theme": "owncloud",
-    "version": "0.1.0",
-    "openIdConnect": {
-      "metadata_url": "https://localhost:9200/.well-known/openid-configuration",
-      "authority": "https://localhost:9200",
-      "client_id": "phoenix",
-      "response_type": "code",
-      "scope": "openid profile email"
-    },
-    "apps": [
-      "files",
-      "draw-io",
-      "pdf-viewer",
-      "markdown-editor",
-      "media-viewer"
-    ]
-  }
-  ```
-
-- Run the OCIS server with necessary configurations
+    yarn run redis-server
+    ```
+3. create a new phoenix config.json file.
```

Comment:
If I understand it correctly, oCIS generates it's own config if the path to an existing one is not specified. What about writing this step as optional?

Reply:
I wasn't aware of that auto-generation I will try it

Reply:
I've deleted the step all-together, also in the auto run